main.cpp

```diff
@@ -74,43 +86,111 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
+ */
+ bool flyAPlane(int maxSpeed, int planeModel = 4, int planeAge = 30)
+ {
+     ignoreUnused(maxSpeed, planeModel, planeAge);
+
+     return {};
+ }
 
 /*
  2)
- */
+ */
+
+ double calculateAverage(int a, int b, int c)
+ {
+     ignoreUnused(a, b, c);
+
+     return {};
+ }
 
 /*
  3)
- */
+ */
+
+ bool washDishes(bool dirtyDishes = true, int numberOfDishes = 56)
+ {
+     ignoreUnused(dirtyDishes, numberOfDishes);
+
+     return {};
+ }
 
 /*
  4)
  */
-
+
+bool playGuitar(bool isRockNRoll = true, int numberOfStrings = 6, int numberOfChords = 3)
+{
+    ignoreUnused(isRockNRoll, numberOfStrings, numberOfChords);
+
+    return {};
+}
 /*
  5)
- */
+ */
+
+bool eatCandy(int numberOfCandy = 4, bool teethAreRotten = true)
+{
+    ignoreUnused(numberOfCandy, teethAreRotten);
+    
+    return {};
+}
 
 /*
  6)
- */
+ */
+
+ int numberOfChordsNeeded(int numberOfBars = 4, bool isPopMusic = true)
+ {
+     ignoreUnused(numberOfBars, isPopMusic);
+
+     return {};
+ }

```

Comment:
This indentation is also incorrect.
<img width="627" alt="Screen Shot 2021-05-25 at 12 00 51 PM" src="https://user-images.githubusercontent.com/2929847/119553879-e57add80-bd50-11eb-9165-1bbb3d5f24a2.png">

Train your eye to see when your code is aligned correctly.
Code readability is very important.   You want to make your code as easy to read as possible, in case someone else will read it.   Compare:
```
 int numberOfChordsNeeded(int numberOfBars=4,bool isPopMusic=true) {ignoreUnused(numberOfBars,isPopMusic);  return {};}
 ```
 with:
 ```
int numberOfChordsNeeded (int numberOfBars = 4, bool isPopMusic = true)
{
    ignoreUnused(numberOfBars, isPopMusic);

    return { };
}
 ```
Which one do you find easier to read?
- [ ] revise